react/ShelfItem.js

```diff
@@ -47,8 +58,11 @@ export default class ShelfItem extends Component {
       <ExtensionPoint
         id="product-summary"
         product={this.normalizeProduct(item)}
+        actionOnClick={this.sendProductClickEvent(item)}
```

Comment:
I think this could be
```suggestion
        actionOnClick={() => this.sendProductClickEvent(item)}
```

Reply:
Good! It resolves why I used a inner function (comment above).

Reply:
🔝

Reply:
When I do like this I actually call sendProductClickEvent twice.